atlp-rwanda/elite-bn-fe
# ch(set-up-redux)

### What does this PR do?
Set up react redux

### How should this be manually tested?
pre-requisite install node and yarn make sure you have the updated versions.

Initial clone the repo

run : yarn install


### Any background context you want to provide?

### What are the relevant pivotal tracker stories?
[#182201516](https://www.pivotaltracker.com/story/show/182201516)
